Adds the environment PYTHONPATH so modules like numpy get found
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/psi4-1.6.1.orig/cmake/TestingMacros.cmake b/psi4-1.6.1/cmake/TestingMacros.cmake
index 07074a7..9498db7 100644
--- a/psi4-1.6.1.orig/cmake/TestingMacros.cmake
+++ b/psi4-1.6.1/cmake/TestingMacros.cmake
@@ -46,7 +46,7 @@ macro(add_regression_test _name _labels)
     )
     set_tests_properties("${_name}"
       PROPERTIES
-        ENVIRONMENT PYTHONPATH=${PSILIB})
+        ENVIRONMENT PYTHONPATH=${PSILIB}:$ENV{PYTHONPATH})
 
     if(labels)
         set_tests_properties(${_name} PROPERTIES LABELS "${labels}")
diff --git a/psi4-1.6.1.orig/tests/pytests/CMakeLists.txt b/psi4-1.6.1/tests/pytests/CMakeLists.txt
index 39a92e3..15b6993 100644
--- a/psi4-1.6.1.orig/tests/pytests/CMakeLists.txt
+++ b/psi4-1.6.1/tests/pytests/CMakeLists.txt
@@ -14,6 +14,6 @@ endif()
 
 # A single command to remake and run the pytest smoke tests
 add_custom_target(pytest
-                  COMMAND PYTHONPATH=${STAGED_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}${PYMOD_INSTALL_LIBDIR}
+	COMMAND PYTHONPATH=${STAGED_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}${PYMOD_INSTALL_LIBDIR}:$ENV{PYTHONPATH}
                           ${Python_EXECUTABLE} -m pytest -rws -v -m smoke --capture=sys ${CMAKE_CURRENT_SOURCE_DIR}
                   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
